.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,72 @@
+# Contributor Code of Conduct
```

Comment:
Why not linking to the documentation like api-platform/api-platform does (https://github.com/api-platform/api-platform/blob/master/.github/CODE_OF_CONDUCT.md - we should update this one too)?

Reply:
We centralize everything here, the other files are deleted.